lib/static.js

```diff
@@ -60,10 +60,16 @@ const copyAssetsFromOptions = async function (markdown) {
 };
 
 const copyAssetsAndWriteFile = async (sourceDir, file, targetDir) => {
-  const markdown = await fs.readFile(path.join(sourceDir, file));
-  const awaits = await copyAssetsFromOptions(markdown);
+  extraOptions = {}
+  const rawMarkdown = await fs.readFile(path.join(sourceDir, file));
+  const { yamlOptions, markdown } = parseYamlFrontMatter(rawMarkdown.toString());
+  const options = Object.assign(getSlideOptions(yamlOptions), extraOptions);
```

Comment:
Looks like the `extraOptions` are not used? Then the assignment and the argument here can be removed.